src/BUILD.gn

```diff
@@ -31,6 +31,10 @@ config("includes") {
   if (chip_device_platform == "linux" || chip_device_platform == "darwin") {
     defines += [ "CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE=${chip_enable_ble}" ]
   }
+
+  if (is_debug) {
+    defines += [ "DEBUG=1" ]
```

Comment:
@mspang I'm not sure if this is the right way to do this.

Reply:
This is already done, but it's done for `NDEBUG`. `DEBUG` is non-standard.

Reply:
Yeah completely agree. Just need to make sure reviewers are aware of which flags exist and which dont. We should just be using NDEBUG everywhere. I'll push another chane tomorrow